docs/monitoring/understanding-dashboards.md

```diff
@@ -0,0 +1,460 @@
+- **otherreject replies**: coutns the number of replies to transaction requests that get dropped for reasons other than paying too little gas
+
+![The tx propagation panel](/images/docs/grafana/tx-propagation.png)
+
+#### Block forwarding
+
+The block forwarding panel counts the announcements and the blocks that the local node receives that it should pass on to its peers. 
+
+#### Transaction fetcher peers
+
+The transaction fetcher peers panel shows how many peers the local node is connected to that can serve requests for transactions. The adjacent transaction fetcher hashes panel shows how many transaction hashes are available for fetching. Three statuses are reported in each panel: Waiting, queuing and fetching.
+
+#### Reorg
+
+The `reorg meter` panel simply counts the blocks added and the blocks removed during chain reorgs. The adjacent `Reorg total` panel shows the total number of reorg executions including both additions and removals.
+
+#### Goroutines
+
+Tracks the total number of active goroutines being used by Geth. Goroutines are lighweight threads managed by the Go runtime, they allow processes to 
+execute concurrently.
+
+![The goroutine panel](/images/docs/grafana/goroutines.png)
+
+#### Eth fetcher filter bodies/headers
+
+Tracks the rate that headers/block bodies arrive from remote peers.
+
+### Database
+
+The database section tracks various metrics related to data storage and i/o in the levelDB and ancients databases.
+
+#### Data rate
+
+measures the rate that data is written to, or read from, the levelDB and ancients databases. Includes:
+
+- **leveldb read**: Rate that data is read to the fast-access levelDB database that stores recent data.
+- **leveldb write**: Rate that data is written to the fast-access levelDB database that stores recent data.
+- **ancient read**: Rate that data is read from the freezer (the database storing older data).
+- **ancient write**: Rate that data is written to the freezer (the database storing older data)
+- **compaction read**: Rate that data is written to the levelDb database while it is being compacted (i.e. free space is reclaimed by deleting uneccessary data)
+- **compaction write**: Rate that data is read from to the levelDB database while it is being compacted (i.e. free space is reclaimed by deleting uneccessary data)
+  
+#### Session totals
+
+Instead of the *rate* that data is read from, and written to, the levelDB and ancients databases (as per `Data rate`), this panel tracks the total amount of data read and written across the entire time Geth is running.
+
+#### Persistent size
+
+This panel shows the amount of data, in GB, in the levelDB and ancients databases.
+
+#### Compaction time, delay and count
+
+These panels show the amount of time spent compacting the levelDB database, duration write operations to the database are delayed due to compaction and the frequency of compaction executions.
```

Comment:
the count of various types of compaction executions.

Reply:
```suggestion
These panels show the amount of time spent compacting the levelDB database, duration write operations to the database are delayed due to compaction and the count of various types of compaction executions.
```